### PR TITLE
Leftover Anti-RightClick: tistory.com

### PR DESCRIPTION
**Pull request by ghost**
### URLs
https://foxeyes.tistory.com/entry/%EC%8B%A4%EC%A0%84%ED%8E%B8-IntelliJ%EC%97%90%EC%84%9C-PluntUML-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0
https://muksteem.tistory.com/18
https://tali.tistory.com/1436

### Source
https://gist.github.com/piquark6046/83c1b4ed197da4b824c5ff73eadb010c

### CODE REVIEW

**Comment by piquark6046** [comment]
### Leftover ( https://github.com/piquark6046/AdguardFilters/commit/661c74a7052c3b0e41858968a6e098619ed75775)
https://fcnotebook.tistory.com/entry/download
https://tali.tistory.com/1436

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1360,6 +1361,7 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+tistory.com#%#AG_onLoad(function() { setTimeout(function() { var GStyle = document.createElement('style'); document.body.appendChild(GStyle); GStyle.type = "text/css"; GStyle.appendChild(document.createTextNode(':not(input):not(textarea):not(code):not(blockquote):not(pre) { user-select: auto!important; -webkit-user-select: auto!important; -webkit-touch-callout: auto!important; }')) }, 800); });
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1364]: I told you earlier, we should avoid using JS. Check these rules.
```
tistory.com#$#html * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
tistory.com#$#html { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/92116/commits/e3e83ac9c1d51fa0654ad712f4126fc1b3fa6c4f

Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt]: It doesn't work with `body`, I gave you rules with `html`.

**Review by zloyden — APPROVED** [review]

**Pull request merged**